packages/sdk/src/oracle/types/interfaces.ts

```diff
@@ -65,6 +74,7 @@ export interface OracleInterface {
   updateFromTransactionReceipt: (receipt: TransactionReceipt) => void;
   getProps: () => Promise<OracleProps>;
   listRequests: () => Requests;
+  parseLog: (log: Log) => ParsedLog;
```

Comment:
We had to expose this function in the oracle interface in order to correctly parse logs from a transaction receipt